parser/tests/Create.test

```diff
@@ -139,6 +139,62 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+# Empty partitioning attribute is not allowed
```

Comment:
@zuyu, do you know how to print the query plan as below for correct statements?

==                                                                                                                                                                                
                                                                                                                                                                                  
CREATE TABLE test (attr INT NULL, attr FLOAT NOT NULL)                                                                                                                            
--                                                                                                                                                                                
CreateTableStatement[relation_name=test]                                                                                                                                          
+-attribute_list=                                                                                                                                                                 
  +-AttributeDefinition[name=attr,type=Int NULL]                                                                                                                                  
  +-AttributeDefinition[name=attr,type=Float]                                                                                                                                     
==